fecgov/openFEC
# Add cand_pty_affiliation and most_recent to real_efile_se_f57_vw

## Summary (required)

- Resolves [#3708](https://github.com/fecgov/openFEC/issues/3708) 

- Add cand_pty_affiliation and most_recent to real_efile_se_f57_vw,  so /schedules/schedule_e/efile/  can 
have the corresponding filters

## How to test the changes locally

- Download the feature branch, and run migration file V0145 (or run invoke create_sample_db) to make sure migration files runs without error.

- An identical view : real_efile_se_f57_vw_tmp_hc was created to demo what the two columns and their data looks like.
